### DISCUSSION TITLE
朋友生病住院了，急需268医药费

### DISCUSSION THREAD

**Discussion by JobNumber-9527** [share121/inter-knot#3292] [其他]
医生说他得了81种病，有没有好心人赞助268元医药费，助他早日打败病魔
<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by tayiie** [comment]
## 嘿，兄弟，你这要帮助他，他可是会化身猿神去挑战神话的啊。
>都病的这么严重了，不如认个父和他儿子抢遗产<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3d389fd2885b008e46d843c552c02bf3_5073438697399152551.png" alt="开拓者 吃瓜脸" width="75" >

**Comment by kaijinti** [comment]
都这样了，要不买三百那个裆位的药吧，好的快一点

**Comment by opqphp** [comment]
节哀
![illust_116799150_20240819_000712](https://github.com/user-attachments/assets/849a6473-103c-4fcf-b68a-e0ab2629e7e2)

**Comment by SKWTEA** [comment]
兄弟，你朋友这病今天就得交钱了吧😮

**Comment by Loxic123** [comment]
我朋友也得了这种病，但是好在我出了328把他治好了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
<img width="749" alt="屏幕截图 2024-08-20 202640" src="https://github.com/user-attachments/assets/9f6022af-50fe-41e0-8dfb-58e7e6d23406">
